certbot/storage.py

```diff
@@ -256,8 +259,26 @@ def __init__(self, config_filename, cli_config):
         self.live_dir = os.path.dirname(self.cert)
 
         self._fix_symlinks()
+        if update_symlinks:
+            self._update_symlinks()
         self._check_symlinks()
 
+    def __str__(self):
+        """Returns a string representation of the object."""
+        # import ipdb; ipdb.set_trace()
+        outdict = {"Lineage": self.lineagename,
+                   "Domains": " ".join(self.names())}
+        return "RenewableCert(%s)" % str(outdict)
```

Comment:
grabbed the date formatting from `openssl x509 -inform pem -noout -text -in`

what's a possible use case for machine-readable? can we default to human-readable, and then also have a machine-readable flag?

I really like the (89 days from now / EXPIRED) idea!